src/prefect/tasks/fivetran/fivetran.py

```diff
@@ -50,11 +54,10 @@ def run(
             - connector_id (str, optional): if provided, will overwrite value provided at init.
             - poll_status_every_n_seconds (int, optional): this task polls the Fivetran API for status,
                 if provided this value will override the default polling time of 15 seconds.
-            - manual (bool, optional): if provided, will overwrite Prefect's changes
-                to the Fivetran connector's schedule, keeping it on Fivetran auto scheduling
-
+            - schedule_type (str, optional): Either manual or auto. If manual (default),
+                connector will run on Prefect's schedule. If auto, connector will remain on Fivetran's
         Returns:
-            - dict: connector_id (str) and succeeded_at (timestamp str)
+            - status (dict): connector_id (str) and succeeded_at (timestamp str)
```

Comment:
It looks like the documentation tests are failing because of the change on this line. It you revert this change then the doc tests should pass.